### PR TITLE
Add Cache SW manual test infrastructure

### LINKED ISSUE #6495: Setup SW for local testing
We should have a manual test page that installs the SW, or at least gives instructions for how to get it working locally.

### PR DESCRIPTION
**Pull request by jridgewell**
Fixes #6495.

/cc @dvoytenko for the extensions script url changes

### CODE REVIEW

**Comment by cramforce** [comment]
Looks good including the move of the extension code into a file with less deps. But don't you need to update some unit tests there?

**Comment by jridgewell** [comment]
> But don't you need to update some unit tests there?

Yup, working on that next.

**Comment by dvoytenko** [comment]
@jridgewell Are there actually any changes to how we calculate extension URL? Or is it just refactoring to a separate module?

**Comment by jridgewell** [comment]
No changes to how they're done, just extracted and added [`calculateEntryPointScriptUrl`](https://github.com/ampproject/amphtml/pull/6532/files#diff-c5ad90666285b32b6075008f9ad89e58R65).

**Comment by jridgewell** [comment]
PTAL

File: test/functional/test-cache-sw-core.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ runner.run('Cache SW', () => {
             'https://cdn.ampproject.org/rtv/123/v0.js');
       });
 
-      it('rewrites v1 to versioned v1', () => {
+      it.skip('rewrites v1 to versioned v1', () => {
```
Comment (cramforce) [inline test/functional/test-cache-sw-core.js]: ?

Comment (jridgewell) [inline test/functional/test-cache-sw-core.js]: Because we don't match `v1` right now, only `v0`.

**Pull request merged**